google-cloud-asset/samples/quickstart.rb

```diff
@@ -149,3 +149,96 @@ def search_all_iam_policies scope: "", query: "", page_size: 0, page_token: ""
   end
   # [END asset_quickstart_search_all_iam_policies]
 end
+
+def build_analysis_query scope:, full_resource_name:
+  resource_selector = {
+    full_resource_name: full_resource_name
+  }
+  options = {
+    expand_groups:      true,
+    output_group_edges: true
+  }
+  query = {
+    scope:             scope,
+    resource_selector: resource_selector,
+    options:           options
+  }
+
+  query
+end
+
+def analyze_iam_policy scope: "", full_resource_name: ""
+  # [START asset_quickstart_analyze_iam_policy]
+  require "google/cloud/asset"
+
+  # scope = 'SCOPE_OF_THE_QUERY'
+  # full_resource_name = 'QUERY_RESOURCE'
+  asset_service = Google::Cloud::Asset.asset_service
+
+  response = asset_service.analyze_iam_policy(
+    analysis_query: build_analysis_query(scope: scope, full_resource_name: full_resource_name)
```

Comment:
You'll need to inline this data structure here rather than calling out to a helper method, even though it results in code duplication across samples. Samples need to be basically complete between the START and END tags so customers can copy/paste and have it work with minimal effort.

Reply:
Then it will result in "too many lines for a method" errors for analyze_iam_policy_longrunning_gcs and analyze_iam_policy_longrunning_bigquery. That's why I extracted them into a helper method. Any idea how to fix the check error?

Reply:
In `google-cloud-asset/rubocop.yml`, add the following:

```
Metrics/MethodLength:
  Exclude:
    - "samples/quickstart.rb"
```

(And please keep the file sorted, i.e. add that clause after the `Metrics/BlockLength` section.)

Reply:
Done